src/cmdlib.sh

```diff
@@ -472,7 +472,7 @@ prepare_git_artifacts() {
         is_dirty="true"
     fi
 
-    tar -C "${gitd}" -czf "${tarball}" --exclude-vcs .
+    tar -C "${gitd}" -cJf "${tarball}" --exclude-vcs .
```

Comment:
For back-compat, I'd suggest just leaving these as gzip for now since they're not covered by the `cosa compress` command. They're not very large anyway.

Reply:
Agreed, should've dropped them before..